python/exercise1/vowel_counter.py

```diff
@@ -2,17 +2,19 @@
 
 def num_vowels(text):
```

Comment:
Should these functions be a single function that takes a target string and counts those in the target string and out of the target string?   
ex:  
```python
def countletters(input_string, match_letters):  
    # your code here
    return (num_matches, num_non_matches)  
```